CHANGELOG_PENDING.md

```diff
@@ -1,3 +1,25 @@
 # Unreleased Changes
 
-## vX.X
+## v0.5.16
+
+### Upcoming Changes
+
+### Features ⚒
+
+#### General
+
+#### Broadcaster
+
+#### Orchestrator
+
+### Bug Fixes 🐞
+
+- [#1782](https://github.com/livepeer/go-livepeer/pull/1782) Fix SegsInFlight data-loss on refreshing O sessions (@darkdragon)
```

Comment:
Fine to not add the PR link here and instead use the [format](https://github.com/livepeer/go-livepeer/blob/master/doc/contributing.md#changelog) described in the changelog portion of the contributing guide because it will be automatically added when we cut a release. So, let's just use the format described there.